src/test/java/com/spotify/ffwd/http/HttpClientTest.java

```diff
@@ -51,6 +51,8 @@
 
     private final List<HttpDiscovery.HostAndPort> servers = new ArrayList<>();
 
+    private final static int SUCCESS = 200;
+
     @Before
     public void setUp() throws Exception {
         mockServerClient.when(pingRequest).respond(response().withStatusCode(200));
```

Comment:
since we have success defined above we can prob use that instead of the '200'

Reply:
Sure,I missed it

Reply:
Done